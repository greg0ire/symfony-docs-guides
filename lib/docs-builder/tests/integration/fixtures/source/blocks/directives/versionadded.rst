
.. versionadded:: 4.1

    The ``foobar`` option was introduced in Symfony 4.1.
