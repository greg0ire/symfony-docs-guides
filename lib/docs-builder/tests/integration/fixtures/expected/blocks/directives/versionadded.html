<div class="admonition admonition-versionadded ">
<p class="admonition-title"><span>4.1</span></p>
<p>The <code translate="no" class="notranslate">foobar</code> option was introduced in Symfony 4.1.</p>
</div>
